tests/Constraints/AbstractConstraintValidatorTest.php

```diff
@@ -103,7 +110,52 @@ protected function createConstraint()
     }
 
     /**
+     * Returns the corresponding IsoCodes library test class to get data to test.
+     *
+     * @return string
+     */
+    protected function getIsoCodesTestClass()
+    {
+        $validatorTestClassTab = explode('\\', get_class($this));
+
+        return str_replace('ValidatorTest', '', end($validatorTestClassTab)).'Test';
+    }
+
+    /**
      * @return string
      */
     abstract protected function getInvalidMessage();
+
+    /**
+     * @param array[] $values
+     *
+     * @return array[]
+     */
+    private function filterEmptyValues($values)
+    {
+        foreach ($values as $v => $value) {
+            if (empty($value[0])) {
+                unset($values[$v]);
+            }
+        }
+
+        return $values;
+    }
+
+    /**
+     * @return AbstractIsoCodeTest
+     */
+    private function getIsoCodesTestInstance()
+    {
+        // Test classes seems not be autoloaded by default.
```

Comment:
In `composer.json`, you can specify how to generate autoload mechanisms for whatever you want (including those test classes I think). Using `autoload-dev` will ensure you don't clutter your production autoload mechanism.